hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenUnfreezeAccountHandlerTest.java

```diff
@@ -236,10 +245,6 @@ void tokenRelNotFound() throws HandleException {
         @Test
         void tokenNotFound() throws HandleException {
             final var token = toPbj(KNOWN_TOKEN_WITH_FREEZE);
-            given(tokenStore.getTokenMeta(token)).willReturn(tokenMetaWithFreezeKey());
-            given(accountStore.getAccountById(ACCOUNT_13257))
-                    .willReturn(Account.newBuilder().accountId(ACCOUNT_13257).build());
-            given(tokenStore.get(token)).willReturn(null);
```

Comment:
Why were these removed?

Reply:
+1

Reply:
The `pureCheck()` method is executed before `tokenStore.getTokenMeta()`, [here](https://github.com/hashgraph/hedera-services/pull/8611/files/5aeeb96b5a4204743e39b3f1e99f335894996d1b#diff-55d2214c26d6ea7346a3bf92e7a3757536c0102dadd73ec12e205e518ed2ae68R65) we can see it. And `pureCheck()` throws `INVALID_TOKEN_ID` exception, [here](https://github.com/hashgraph/hedera-services/pull/8611/files/5aeeb96b5a4204743e39b3f1e99f335894996d1b#diff-55d2214c26d6ea7346a3bf92e7a3757536c0102dadd73ec12e205e518ed2ae68R100). You can track easily who did these changes in develop (adding pure checks). The test started to fail once I updated my branch with develop.

Reply:
This test is testing that, even though `pureCheck()` passes, we still need to be able to get the token from state (i.e. `tokenStore.get(token)`. Removing these lines invalidates the test

Reply:
Oh, got it :pray:. Just reverted my changes.

Reply:
Thank you!